src/utils/shareReply.ts

```diff
@@ -0,0 +1,19 @@
+import templite, { Values } from 'templite';
+import { log } from './log';
+import messages from './messages.json';
+
+export function share(replyName: string, values?: Values): string {
```

Comment:
I think this should be renamed to something else. When you were talking about this in the discord server I thought you were saying about the "share" message (pc-ios/android)

I would name it `t`, or maybe `getMessage`.